dy_volumes_cleanup/cli.py

```diff
@@ -63,7 +60,12 @@ async def async_entrypoint(
 
             await delete_volume(client, volume_name)
             typer.echo(f"Removed docker volume: '{volume_name}'")
-
+            numberOfCleanedUpZombies = numberOfCleanedUpZombies + 1
```

Comment:
you can also use the shorthand version
```suggestion
            cleaned_up_volumes_count += 1
```